recipes/pam.rb

```diff
@@ -48,8 +48,14 @@
     end
 
     # configure passwdqc via central module:
+    template_cookbook = if node['os-hardening']['auth']['pam'].attribute?('passwdqc') &&
```

Comment:
Even if this is really a proper way to do that, we have all this attributes configured in the default level. So its safe to assume that they exist, I would suggest to avoid the checks:

```ruby
template_cookbook = node['os-hardening']['auth']['pam']['passwdqc']['template_cookbook']
```

Reply:
Want to ask for clarification - do you want me to set the default cookbook name used for the attribute in the attributes file? Or is it sufficient to simply remove the check for the `['pam']` attribute but leave the check for the passwdqc and other attributes?

Reply:
Good idea! We can specify the default cookbook name and avoid additional variables and conditions.